WordPress/UITests/Flows/LoginFlow.swift

```diff
@@ -24,13 +24,13 @@ class LoginFlow {
 
     // Login with WP site via Site Address.
     @discardableResult
-    static func login(siteUrl: String, email: String, password: String) throws -> MySiteScreen {
+    static func login(siteUrl: String, email: String, password: String, title: String? = nil) throws -> MySiteScreen {
```

Comment:
From looking at the `login` function name, it's not automatically clear that `title` is the site title (which is the case for `continueWithSelectedSite`, where `title` param name is intuitive). WDYT of going for `selectedSiteTitle` as it's done two functions above?